pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -2173,6 +2173,15 @@ self: super: {
     testTarget = "tests";
   }) super.conduit-aeson;
 
+  # Ships a broken Setup.hs
+  # https://github.com/lehins/FailT/issues/1
+  FailT = overrideCabal (drv: {
+    postPatch = ''
+      ${drv.postPatch or ""}
+      rm Setup.hs
```

Comment:
The issue seems to indicate that this has already been released to Hackage.  Is this necessary?  Or do we need to update our Hackage pin first?

Reply:
Yes, my plan was to just drop the broken flag in the next iteration.

Reply:
@cdepillabout 
You are correct.  I misread the pin update time, it was two hours too early and that is why my nix build of `FailT` failed.
Not sure how I missed the last comment in the aforementioned issue that said it was uploaded to Hackage.

Will be more careful next time.